tmk_core/protocol/lufa/adafruit_ble.cpp

```diff
@@ -29,6 +29,11 @@
 #define SAMPLE_BATTERY
 #define ConnectionUpdateInterval 1000 /* milliseconds */
 
+#ifdef SAMPLE_BATTERY
```

Comment:
Given the above code, it will never not be defined here.
```suggestion
```

Reply:
That's true ... but I'm trying to touch the existing code as little as possible.